sjb/config/common/test_cases/origin_release_with_service_catalog.yml

```diff
@@ -0,0 +1,24 @@
+---
+parent: 'common/test_cases/origin_release.yml'
+overrides:
+  provision:
+    os: "rhel"
+    stage: "build"
+    provider: "aws"
+extensions:
+  sync:
+    - "openshift,service-catalog=master"
+  actions:
+    - type: "script"
+      title: "determine the release commit for service catalog image and version for rpms"
+      repository: "origin"
+      script: |-
+        catalog_repo="/data/src/github.com/openshift/service-catalog/"
+        git log -1 --pretty=%h >> "${catalog_repo}/ORIGIN_COMMIT"
+    - type: "script"
+      title: "build the service catalog container image"
+      repository: "service-catalog"
+      timeout: 3600
+      script: |-
+        make build-images
+        docker tag openshift/origin-service-catalog:latest "openshift/origin-service-catalog:$( cat ./ORIGIN_COMMIT )"
```

Comment:
Install origin playbook relies on the service catalog, which is no longer in origin. Sync in the new service catalog and rebuild the image.

Reply:
yep lines up with what was discussed

Reply:
so there's no existing job we can extend from that already builds/tags the service catalog appropriately?  how are the other e2e jobs running?